neworigins_v2/world.cpp

```diff
@@ -373,6 +373,44 @@ const char *AGetNameString(int name)
 	return "Error";
 }
 
+// void WriteMapAsJson(ARegionArray* map) {
```

Comment:
Can be deleted?

Reply:
I will delete when all changes will be added, i.e. when we will agree that the map look good :)
I helps to debug map a alot